tests/TinkerTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Spatie\WebTinker\Tests;
+
+use Spatie\WebTinker\Tinker;
+
+class TinkerTest extends TestCase
+{
+    /** @test */
+    public function remove_c_style_single_line_comments()
```

Comment:
Could you rename the names of the tests to an sentence.

Example: `it_removes_c_style_single_line_comments()`